cli/root.go

```diff
@@ -34,6 +34,10 @@ var RootCommand = &cobra.Command{
 			"version",
 		}
 
+		if valid, prefixValidationError := util.IsValidTagPrefix(rootPrefixFlag); !valid {
```

Comment:
Just a small change here, please:

```suggestion
		if isValid, prefixValidationError := util.IsValidTagPrefix(rootPrefixFlag); !isValid {
```